btcjson/walletsvrresults.go

```diff
@@ -159,3 +159,19 @@ type GetBestBlockResult struct {
 	Hash   string `json:"hash"`
 	Height int32  `json:"height"`
 }
+
+type importMultiError struct {
+	Code    int    `json:"code"`
+	Message string `json:"message"`
+}
+
+// ImportMultiResults is a slice that models the result of the importmulti command.
+//
+// Each item in the slice contains the execution result corresponding to the input
+// requests of type btcjson.ImportMultiRequest, passed to the ImportMulti[Async]
+// function.
+type ImportMultiResults []struct {
+	Success  bool              `json:"success"`
+	Error    *importMultiError `json:"error,omitempty"`
```

Comment:
Looking at Bitcoin Core implementation this `error` element is the standard JSON-RPC Error `btcjson.RPCError`

Reply:
Fixed. 👍